flagsmith-jira-app/src/IssueFlagPanel.tsx

```diff
@@ -126,46 +157,23 @@ const IssueFlagTable = ({
                     <Text>Last updated</Text>
                   </Cell>
                 </Head>
-                {environments.map((environment) => {
-                  const environmentFlags = flags[String(environment.id)] ?? [];
-                  // get the default state for this environment
-                  const flag = environmentFlags.find(
-                    (each) =>
-                      String(each.feature) === String(featureId) &&
-                      each.feature_segment === null &&
-                      each.identity === null,
-                  );
-                  if (!flag) return null;
-                  const value: Partial<FeatureStateValue> = flag.feature_state_value ?? {};
-                  // count variations/overrides
-                  const variations = flag.multivariate_feature_state_values.length;
-                  const segments = environmentFlags.filter(
-                    (each) =>
-                      String(each.feature) === String(featureId) && each.feature_segment !== null,
-                  ).length;
-                  const identities = environmentFlags.filter(
-                    (each) => String(each.feature) === String(featureId) && each.identity !== null,
-                  ).length;
+                {environmentFlag?.environments.map((environment: any) => {
+                  if (!environment) return null;
+                  // count overrides
+                  const segments = environment.feature_segment || 0;
+                  const identities = environment.identity || 0;
```

Comment:
I don't understand this code. What are `environment.feature_segment` and `environment.identity` ?

Reply:
This was already corrected in the latest version.